src/app/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import styles from './Tabs.module.css';
+
+function getActiveClass(pathname: string) {
+  const location = useLocation();
+  if (!location.pathname.includes(pathname)) {
+    return;
+  }
+  return styles.tabs__active;
+}
+
+function Tabs(): JSX.Element {
```

Comment:
This is not "Tabs", it's a navigation with 2 links.
Examples for tabs:
https://material-ui.com/components/tabs/
https://react.school/ui/tabs